src/modules/alertmanager/CriticalAlertsService.ts

```diff
@@ -0,0 +1,65 @@
+import { inject, injectable }                    from 'inversify';
+import { ILogger }                               from '../logger/ILogger';
+import { Environment }                           from '../environment/Environment';
+import got                                       from 'got';
+import { AlertRequestBody, PreparedToSendAlert } from './alerts/BasicAlert';
+import { CriticalNegativeDelta }                 from './alerts/CriticalNegativeDelta';
+import { CriticalMissedProposes }                from './alerts/CriticalMissedProposes';
+import { CriticalMissedAttestations }            from './alerts/CriticalMissedAttestations';
+import { ClickhouseStorage }                     from '../storage/ClickhouseStorage';
+
+type SentAlerts = { [alertname: string]: PreparedToSendAlert };
+
+export const sentAlerts: SentAlerts = {};
+
+@injectable()
+export class CriticalAlertsService {
+
+  private readonly baseUrl;
+
+  public constructor(
+    @inject(ILogger) protected logger: ILogger,
+    @inject(Environment) protected env: Environment,
+    @inject(ClickhouseStorage) protected storage: ClickhouseStorage,
+  ) {
+    this.baseUrl = this.env.ALERTMANAGER_URL ?? '';
+  }
+
+  private get alerts() {
+    return [
+      new CriticalNegativeDelta(this.env, this.storage),
+      new CriticalMissedProposes(this.env, this.storage),
+      new CriticalMissedAttestations(this.env, this.storage)
+    ]
+  }
+
+  public async sendCriticalAlerts(bySlot: bigint) {
+    if (!this.baseUrl) {
+      this.logger.warn(`Env var 'ALERTMANAGER_URL' is not set. Unable to send critical alerts`)
+      return;
```

Comment:
May be it's better to fail at init phase, since it is critical alerts?

Reply:
Alertmanager is optional for this project. User decides to receive alerts or not

Reply:
So maybe we can load the module itself by condition?

Reply:
I'm not sure if that makes sense. Soon we will get rid of `inversify` and use `NestJS` and maybe there will be a good solution.

I propose to leave this solution, it provides a quick exit from class method and that is enough for now